module/documents/active-effect.mjs

```diff
@@ -647,7 +647,7 @@ export default class ActiveEffect5e extends ActiveEffect {
       value: app.document.getFlag("dnd5e", "riders.statuses") ?? [],
       options: CONFIG.statusEffects.map(se => ({ value: se.id, label: se.name }))
     }).outerHTML;
-    html.querySelector("[data-tab=details] > .form-group:last-child").insertAdjacentHTML("afterend", element);
+    html.querySelector("[data-tab=details] > .form-group:last-of-type").insertAdjacentHTML("afterend", element);
```

Comment:
We can use the `has` selector to find the `form-group` that contains the statuses element. We can also add some optional chaining in case it is removed and switch to the `after` command for a bit of simplification (also removes the need for the `outerHTML` above.

```suggestion
    html.querySelector("[data-tab=details] > .form-group:has([name=statuses])")?.after(element);
```